src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/partials/NoReputationError.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { useReputationValidation } from '~v5/common/ActionSidebar/hooks/index.ts';
+import Link from '~v5/shared/Link/index.ts';
+import NotificationBanner from '~v5/shared/NotificationBanner/index.ts';
+
+const displayName =
+  'v5.common.ActionsContent.partials.ActionSidebarContent.partials.NoReputationError';
+
+const MSG = defineMessages({
+  noReputationErrorTitle: {
+    id: `${displayName}.noReputationErrorTitle`,
+    defaultMessage: 'There is no reputation in this team yet',
+  },
+  noReputationError: {
+    id: `${displayName}.noReputationError`,
+    defaultMessage:
+      'If you have the necessary permissions you can bypass the governance process.',
+  },
+});
+
+const NoReputationError = () => {
+  const { formatMessage } = useIntl();
+
+  const { noReputationError } = useReputationValidation();
+
+  if (!noReputationError) {
+    return null;
+  }
+
+  return (
+    <div className="mt-6">
+      <NotificationBanner
+        status="warning"
+        icon="warning-circle"
+        description={formatMessage(MSG.noReputationError)}
+        callToAction={
+          <Link to="https://docs.colony.io/use/reputation">
```

Comment:
This needs to be URL constant

Reply:
These aren't my changes and I want to avoid bloating this PR any more with unrelated fixes

Reply:
In that case, just make sure these changes aren't lost, and we keep track of them